src/components/axes/timeAxis.ts

```diff
@@ -280,12 +287,19 @@ export module Axis {
 
     protected _setup() {
       super._setup();
+      this._createDomElements();
+    }
+
+    private _createDomElements() {
+      d3.selectAll(".time-axis-tier").remove();
```

Comment:
the function is somewhat misnamed.  You're creating Dom elements by first removing the existing ones.  Is there a way that we can keep this logic being _setup and then having the setter for axis configuration logic route to here?